tf-modules/cumulus-rds-tf/outputs.tf

```
@@ -0,0 +1,16 @@
+output "security_group_id" {
+  value = aws_security_group.rds_cluster_access.id
+}
+
+output "rds_endpoint" {
+  value = aws_rds_cluster.core_team_cluster.endpoint
+}
+
+output "secret_arn" {
```

```suggestion
output "admin_access_secret_arn" {
```

And same for the secret version below

---

The output from this module is *implied* to be that though, as it's the entire purpose of the module is to setup the cluster/admin user.   Putting 'admin' on everything in this module seems redundant, if perhaps more explicit in integration. 

If you feel strongly about it, it's not a hill I want to die on either.

---

I guess it's not immediately obvious to me based on the module name or its purpose of setting up an RDS cluster that the `secret_arn` output is **necessarily** a secret containing admin access credentials. The `secret_arn` output from the module creating an RDS cluster could still be a secret containing almost anything, I think. Not a hill I'm going to die on either

---

Hmm, that's fair.  And "secret_arn" is pretty much SECRET THING.   Not terribly descriptive.  I see your point.

How about: 919095f70